pstatus-report-sink-ktor/src/main/kotlin/gov/cdc/ocio/processingstatusapi/models/DeadLetterReport.kt

```diff
@@ -25,5 +24,7 @@ import java.util.*
 
     var deadLetterReasons: List<String>? = null
 
+    var deadLetterReason: String? = null
+
```

Comment:
Let's keep the deadLetterReasons and get rid of deadLetterReason.  When you only have one entry it's okay to only have one entry - just add it as listOf(deadLetterReason).